Documentation/DocuBlocks/Rest/Indexes/post_api_index_inverted.md

```diff
@@ -87,14 +87,22 @@ Default: the value defined by the top-level `trackListPositions` option.
 @RESTSTRUCT{cache,post_api_index_inverted_fields,boolean,optional,}
 Enable this option to always cache the field normalization values in memory
 for this specific field. This can improve the performance of scoring and
-ranking queries.
+ranking queries. Otherwise, these values are memory-mapped and it is up to the
+operating system to load them from disk into memory and to evict them from memory.
```

Comment:
Also here we could mention fields processed with Geo analyzer. There is also auxilarry data to run geo SEARCH and also could be cached.